### PR TITLE
starkmatt/c03-docker05

### PR DESCRIPTION
**Pull request by starkmatt**
# Description

Exercise c03-Docker05

Student Name: Matt G

Pulled and ran the images as expected!

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Ran the existing command from my inital containers created in Exercise 2 but pulled it from my Docker Hub repoistry 
+I ensued that I was logged into Docker CLI and ran the following
+
+
+##Pulling images
+1 = docker pull starkmatt/exercise02:v1
+2 = docker pull starkmatt/exercise02:v2
+
+
+➜  starkmatt git:(starkmatt/c03-docker05) ✗ docker pull starkmatt/exercise02:v1
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt]: it;s fine to pull the images before, but docker will realise it doesn't have it locally and will download when you execute the `docker run` command.

File: classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+a27b80e2bf04: Already exists 
+2c005fed9a91: Already exists 
+Digest: sha256:b8d253f4711797adcff1d6d1e367fb07861edb737492de58472d6dfaa1b9bf68
+Status: Downloaded newer image for starkmatt/exercise02:v2
+
+
+
+## Running images
+
+➜  starkmatt git:(starkmatt/c03-docker05) ✗ docker run -dp 8081:80 docker pull starkmatt/exercise02:v1
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt]: this command doesn't seem to be right.

File: classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt (hunk 1 above)
Comment (starkmatt) [reply classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt]: I see,
Thanks!

File: classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Status: Downloaded newer image for starkmatt/exercise02:v2
+
+
+
+## Running images
+
+➜  starkmatt git:(starkmatt/c03-docker05) ✗ docker run -dp 8081:80 starkmatt/exercise02:v1
+c28514e6680d64c065b0b0795d7d52af6bf8a71495c9b46ad70288b803a86686
+
+➜  starkmatt git:(starkmatt/c03-docker05) ✗ docker run -dp 8082:80 starkmatt/exercise02:v2
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt]: you're not mapping anything here. shouldn't it map a specific folder from your local?

Comment (starkmatt) [reply classes/03class/exercises/c03-docker05/starkmatt/c03-docker05-01.txt]: Yes you are right,
Let me change that!

**Pull request merged**